bin/pull-from-dockerhub.sh

```diff
@@ -41,3 +41,4 @@ docker pull $CCP_IMAGE_PREFIX/crunchy-upgrade:$CCP_IMAGE_TAG
 docker pull $CCP_IMAGE_PREFIX/crunchy-scheduler:$CCP_IMAGE_TAG
 docker pull $CCP_IMAGE_PREFIX/crunchy-pgbench:$CCP_IMAGE_TAG
 docker pull $CCP_IMAGE_PREFIX/crunchy-pgbasebackup-restore:$CCP_IMAGE_TAG
+docker pull $CCP_IMAGE_PREFIX/crunchy-postgres-ha:$CCP_IMAGE_TAG
```

Comment:
@andrewlecuyer `crunchy-postgres-gis-ha` too?

Reply:
`crunchy-postgres-gis-ha` is now included